requirements.txt

```diff
@@ -2,4 +2,4 @@ paho-mqtt
 PyYAML
 enum34
 cerberus
-
+OPi.GPIO
```

Comment:
This should not be in the main requirements - each of the modules have their own requirements which you've already set in `modules/orangepi.py`. If they don't exist then they're installed at run-time. It's done this way so that we only have to install the requirements for the modules we're using.